service/controller/v10/context/awsservice/context.go

```diff
@@ -0,0 +1,26 @@
+package awsservice
```

Comment:
Instead of all this context packages, can't we have one struct which holds all of those and store it inside the context?

Reply:
Yes good point, I will refactor that as a single `context` package and storing all those together.